### PR TITLE
add Masaryk University

### PR DESCRIPTION
**Pull request by melanger**
Add MUNI Unified Login which supports TOTP and WebAuthn.

### CODE REVIEW

File: entries/i/id.muni.cz.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "ics.id.muni.cz"
+    ],
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://it.muni.cz/en/services/jednotne-prihlaseni-na-muni/navody/how-to-set-up-multi-factor-authentication",
+    "recovery": "https://it.muni.cz/en/forms/report-a-problem-with-single-sign-on",
+    "keywords": [
+      "education",
```
Comment (Carlgo11) [inline entries/i/id.muni.cz.json]: higher education should be under universities only.
```suggestion
```

File: entries/i/id.muni.cz.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "MUNI Unified Login": {
+    "domain": "id.muni.cz",
+    "additional-domains": [
+      "ics.id.muni.cz"
+    ],
```
Comment (Carlgo11) [inline entries/i/id.muni.cz.json]: We don't use subdomains. Please rename the files to `muni.cz.json` and `muni.cz.svg`
```suggestion
    "domain": "muni.cz",
```

File: img/i/id.muni.cz.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="61.147" height="97.067"><path d="M344.238 728.012h114.395V0H344.238Zm-42.636 0h38.476L269.355 0h-38.476zm-184.082 0h38.476L226.719 0h-38.477Zm-117.52 0h113.359V0H0v728.012" style="fill:#2d4ba6;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 97.067)"/></svg>
```
Comment (Carlgo11) [inline img/i/id.muni.cz.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36.6 36.6"><path fill="#0000dc" d="M9.92 5v26.6h4.14V5Zm4.29 0 2.59 26.6h1.4L15.62 5Zm6.73 0-2.58 26.6h1.4L22.35 5Zm1.56 0v26.6h4.18V5Z"/></svg>
```

File: entries/i/id.muni.cz.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "MUNI Unified Login": {
```
Comment (Carlgo11) [inline entries/i/id.muni.cz.json]: Entries don't list the authentication systems but the websites using them. Here that would be Mazarykova univerzita or Masaryk University.
```suggestion
  "Mazarykova univerzita": {
```

File: entries/i/id.muni.cz.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "domain": "id.muni.cz",
+    "additional-domains": [
+      "ics.id.muni.cz"
+    ],
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://it.muni.cz/en/services/jednotne-prihlaseni-na-muni/navody/how-to-set-up-multi-factor-authentication",
+    "recovery": "https://it.muni.cz/en/forms/report-a-problem-with-single-sign-on",
```
Comment (Carlgo11) [inline entries/i/id.muni.cz.json]: A region element would be appropriate here.
```suggestion
    "recovery": "https://it.muni.cz/en/forms/report-a-problem-with-single-sign-on",
    "regions": [
      "cz"
    ],
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/i/id.muni.cz.json (hunk 2 above)
Comment (melanger) [reply entries/i/id.muni.cz.json]: There are subdomain files, like [id.atlassian.com.json](https://github.com/2factorauth/twofactorauth/blob/master/entries/i/id.atlassian.com.json) and [id.nintendo.net.json](https://github.com/2factorauth/twofactorauth/blob/master/entries/i/id.nintendo.net.json). Should I change mine to `muni.cz` or leave it as it is?

File: entries/i/id.muni.cz.json (hunk 5 above)
Comment (melanger) [reply entries/i/id.muni.cz.json]: [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) says:
```
If a site (with or without 2FA) is only available in certain countries - for example a government site - you can note this with the regions field
```
but the site is available to people world wide and provides access to services from many countries. Should I still add the region?

Comment (Carlgo11) [reply entries/i/id.muni.cz.json]: Yes please.

File: entries/i/id.muni.cz.json (hunk 2 above)
Comment (Carlgo11) [reply entries/i/id.muni.cz.json]: I should have said, we don't use subdomains _any longer_. Apparently some have slipped through.

**Comment by melanger** [comment]
@Carlgo11 I have applied all the suggestions.

File: img/m/muni.cz.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36.6 36.6"><path fill="#0000dc" d="M9.92 5v26.6h4.14V5Zm4.29 0 2.59 26.6h1.4L15.62 5Zm6.73 0-2.58 26.6h1.4L22.35 5Zm1.56 0v26.6h4.18V5Z"/></svg>
```
Comment (phallobst) [inline img/m/muni.cz.svg]: Here's the logo without padding:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16.76 26.6"><path fill="#0000dc" d="M0 0v26.6h4.14V0Zm4.29 0 2.59 26.6h1.4L5.7 0Zm6.73 0L8.44 26.6h1.4L12.43 0Zm1.56 0v26.6h4.18V0Z"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**